mmrazor/models/mutators/channel_mutator/channel_mutator.py

```diff
@@ -111,7 +111,7 @@ def prepare_from_supernet(self, supernet: Module) -> None:
             raise NotImplementedError()
 
         for unit in units:
-            unit.prepare_for_pruning(supernet)
+            # unit.prepare_for_pruning(supernet)
```

Comment:
unit.prepare_for_pruning(supernet) is esential for the ways using tracer and config. Please copy and paste unit.prepare_for_pruning(supernet) after _prepare_from_tracer and _prepare_from_predefined_model

Reply:
thx, will do